edb/pgsql/delta.py

```diff
@@ -2324,7 +2324,7 @@ def adjust_pointer_storage(self, pointer, schema, orig_schema, context):
 
                         alter_type = dbops.AlterTableAlterColumnType(
                             old_ptr_stor_info.column_name,
-                            common.qname(*new_type))
+                            common.quote_type(*new_type))
```

Comment:
Remove the unpacking here. `quote_type` takes a tuple as its argument

Reply:
Done